### ISSUE TITLE
Allow non-exclusive token voting

### ISSUE THREAD

**Issue by danfinlay** [wslyvh/tokenlog#53]
Current basic token voting allows each token to be applied to vote only once.

An alternative approach (also very simple) would allow a user to vote up to their token holdings on every single issue.

I think this would be nice because it's much harder for a user to mentally compensate & adjust your weighting of every other issue every time you vote, and it's not like these tokens are getting burned/contributed on completion, so it doesn't seem like any functionality is lost.

**Comment by danfinlay** [comment]
I realize this is almost a duplicate with  #4, except the UX is different: Voting `0-${your_balance}` vs `0-1`.

**Comment by wslyvh** [comment]
Currently, your tokens remain 'locked' until an issue is closed. They idea for doing that is people wouldn't just vote on every new issue. That would make still make prioritization difficult, especially if the team already started working/implementing it.
But maybe that isn't so much of an issue.

Either way, I'm very much in favor of allowing projects to select their own voting mechanisms.
